src/messages/messages.json

```diff
@@ -4,7 +4,7 @@
     "exit_code": "SOAtest run finished with code {0}",
     "failed_run_non_zero": "SOAtest run failed with a non-zero exit code: {0}",
     "wrk_dir_not_exist": "Working directory does not exist: {0}",
-    "found_xml_report": "Found Parasoft SOAtest XML report file: {0}",
+    "found_latest_xml_report": "Found latest Parasoft SOAtest XML report file: {0}",
```

Comment:
Please revert back to the old message since I don't think user need to know about this.
@rwang-parasoft What do you think?

Reply:
Yes, I think old message should be better. But maybe we can give some debug messages to recoding the process of finding.

Reply:
I think we have enough messages. Do you mean to change some of them to debug messages instead of info messages?